core-ui/src/components/Predefined/Details/Sidecars/Sidecars.details.js

```diff
@@ -19,6 +19,7 @@ export const SidecarsDetails = ({ DefaultRenderer, ...otherParams }) => {
     <WorkloadSelector
       resource={sidecar}
       labels={sidecar.spec?.workloadSelector?.labels}
+      title={t('workload-selector.title')}
```

Comment:
Once we add a default title in the `WorkloadSelector` component, this line won't be needed